stan/math/opencl/kernel_generator/matrix_cl_conversion.h

```diff
@@ -0,0 +1,29 @@
+#ifndef STAN_MATH_OPENCL_KERNEL_GENERATOR_MATRIX_CL_CONVERSION_H
+#define STAN_MATH_OPENCL_KERNEL_GENERATOR_MATRIX_CL_CONVERSION_H
```

Comment:
Why is this file .h?

Reply:
Due to some editior default i did not notice. Fixed.